ci/rpm_build_success.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Script to be run on successful RPM build
+
+set -uex
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/parse_ci_envs.sh"
+if [ -e "${ci_envs}" ]; then
+  # shellcheck source ci/parse_ci_envs.sh
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "disable". Fix any mentioned problems and try again. [SC1072]